corehq/apps/userreports/reports/view.py

```diff
@@ -398,6 +399,11 @@ def get_ajax(self, params):
         }
         if total_row is not None:
             json_response["total_row"] = total_row
+        if data_source.data_source.config.backend_id == UCR_LABORATORY_BACKEND:
+            compare_ucr_dbs.delay(
+                self.domain, self.report_config_id, self.filter_values,
+                sort_column, sort_order, params
+            )
```

Comment:
Based on a quick grep of the nginx logs it's around 1k requests/day. I implemented it as a backend so we could possibly slowly scale up these, since it would only happen on data sources that we explicitly set. What do you have in mind for deciding when to do these?